inst/include/Rcpp/exceptions.h

```diff
@@ -89,6 +89,48 @@ namespace Rcpp {
             file_io_error("file already exists", file) {}	// #nocov end
     };
 
+    #define RCPP_ADVANCED_EXCEPTION_CLASS(__CLASS__, __WHAT__)                                                                                                                                                     \
```

Comment:
I'm going to push an updated version of the `tinyformat.h`. It last received love in Aug 29, 2015. I need a fix released in [March 2016](https://github.com/c42f/tinyformat/commit/5abc03b35457f1af1b0c7257129017ff5a82e504) to quiet a `[-Wuninitialized]` complaint. After that, everything else should work.